### PR TITLE
feat: Support for Multiple Inbound Transports in Aries Rest binding

### LINKED ISSUE #1163: Agent REST build update to support multiple inbound transport providers
Update agent rest build to support multiple transport providers #1124

### PR DESCRIPTION
**Pull request by rolsonquadras**
Removed --inbound-transport option and updated --inbound-host and --inbound-host-external option to take 'scheme@url' format to support multiple schemes.

closes #1163 

Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

File: test/bdd/fixtures/agent-rest/docker-compose.yml [hunk 1]
```diff
@@ -12,8 +12,8 @@ services:
     image: ${AGENT_REST_IMAGE}:${AGENT_REST_IMAGE_TAG}
     environment:
       - ARIESD_API_HOST=${ALICE_HOST}:${ALICE_API_PORT}
-      - ARIESD_INBOUND_HOST=${ALICE_HOST}:${ALICE_INBOUND_PORT}
-      - ARIESD_INBOUND_HOST_EXTERNAL=http://alice.aries.example.com:${ALICE_INBOUND_PORT}
+      - ARIESD_INBOUND_HOST=http@${ALICE_HOST}:${ALICE_INBOUND_PORT}
```
Comment (sudeshrshetty) [inline test/bdd/fixtures/agent-rest/docker-compose.yml]: minor: schema can be moved to `.env` file

Comment (rolsonquadras) [reply test/bdd/fixtures/agent-rest/docker-compose.yml]: [Moved](https://github.com/hyperledger/aries-framework-go/compare/e5a6403471837f561ea71f34e3c06e00a9f6d907..1f9c8c18c2255f071b6aa295fbde2d85e23aa7d2) to .env file.

File: cmd/aries-agent-rest/startcmd/start_test.go [hunk 2]
```diff
@@ -241,9 +250,17 @@ func TestStartCmdWithMissingHostArg(t *testing.T) {
...
+	parameters := &agentParameters{
+		server:               &mockServer{},
+		host:                 "",
+		inboundHostInternals: []string{randomURL()},
+		inboundHostExternals: []string{},
+		dbPath:               "",
+		defaultLabel:         "",
+		webhookURLs:          []string{},
+		httpResolvers:        []string{},
+		outboundTransports:   []string{},
```
Comment (soluchok) [inline cmd/aries-agent-rest/startcmd/start_test.go]: Why do we need these empty values? I would remove it.

Comment (troyronda) [reply cmd/aries-agent-rest/startcmd/start_test.go]: also on L287

Comment (rolsonquadras) [reply cmd/aries-agent-rest/startcmd/start_test.go]: [agreed] - that was an existing code and I didn't focus on it. [Removed](https://github.com/hyperledger/aries-framework-go/compare/1f9c8c18c2255f071b6aa295fbde2d85e23aa7d2..ca59224205e2d944dea6c38a431bfa561f3e7e2c) now.

**Pull request merged**